Fix small issues (#92)
vtex-apps/category-menu: docs/README.md

}
```

| Prop name            | Type           | Description                                                                                     | Default Value |
| -------------------- | -------------- | ----------------------------------------------------------------------------------------------- | ------------- |

> | `showAllDepartments` | `Boolean`      | Shows all departments category in menu                                                          | `true`        |
> | `menuDisposition`    | `Enum`         | Indicates the disposition of the menu on the screen. Possible values: `left`, `center`, `right` | `center`      |
> | `showSubcategories`  | `Boolean`      | Decides if the subcategories will be displayed                                                  | `true`        |
> | `departments`        | `Array(items)` | List of departments `items` to be displayed on the menu                                         | `[]`          |
> | `mobileMode`         | `Boolean`      | Use this as `true` to render the category menu in a sidebar                                      | `false`       |

Reviewer: ```suggestion
| `showAllDepartments` | `boolean`    | Whether all departments will be displayed on the menu (`true`) or not (`false`).          | `true`        |
| `departments`        | `array` | List of departments to be displayed on the menu.                                         | `undefined`          |
| `showSubcategories`  | `boolean`    | Whether the subcategories will be displayed on the menu (`true`) or not (`false`).      |  `true`     |
| `menuDisposition`    | `enum`         | Menu position on the screen. Possible values are: `left`, `center` or `right`. | `center`      |
| `mobileMode`         | `boolean`      | Whether the category menu will be sidebar rendered (`true`) or not (`false`).      | `false`       |
```
